rag-examples/pom.xml

```diff
@@ -88,6 +88,28 @@
             <version>1.5.12</version>
         </dependency>
 
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-google-ai-gemini</artifactId>
```

Comment:
I'd probably replace the open-ai library with this.

Reply:
Unfortunately there are other projects utilising this dependency. So the project fails to build since there will be errors! 
Unless if we want to delete all the other example folders of course

Reply:
Ah! Right!